dkc/core/models/file.py

```diff
@@ -62,5 +62,15 @@ def clean(self) -> None:
 
 @receiver(models.signals.pre_save, sender=File)
 def _file_pre_save(sender: Type[File], instance: File, **kwargs):
-    # TODO this is where we might handle quotas
-    instance.size = instance.blob.size
+    # TODO if we allow changing a file's blob, we also need to update the quota
+
+    if not instance.pk:
```

Comment:
The third argument passed to this hook is `created: bool`, which we should use instead of this check.

Reply:
`pre_save` doesn't have that argument: https://docs.djangoproject.com/en/3.1/ref/signals/#pre-save

Only `post_save` has it.

Reply:
Oops, forgot this was pre_save.